test/unit/edgeware/testUtil.ts

```diff
@@ -183,8 +186,11 @@ export function constructFakeApi(
       },
       treasury: {
         proposals,
+        bounties,
```

Comment:
These tests will need to be moved to derives as well once the enricher logic is changed.